deployment/terraform/create.go

```diff
@@ -74,23 +72,23 @@ func (t *Terraform) Create() error {
 			return fmt.Errorf("binary path %s has to be a regular file", binaryPath)
 		}
 
-		t.config.DeploymentConfiguration.MattermostDownloadURL = latestReleaseURL
+		t.config.MattermostDownloadURL = latestReleaseURL
 		uploadBinary = true
 	}
 
 	err = t.runCommand(nil, "apply",
-		"-var", fmt.Sprintf("cluster_name=%s", t.config.DeploymentConfiguration.ClusterName),
-		"-var", fmt.Sprintf("app_instance_count=%d", t.config.DeploymentConfiguration.AppInstanceCount),
-		"-var", fmt.Sprintf("ssh_public_key=%s", t.config.DeploymentConfiguration.SSHPublicKey),
-		"-var", fmt.Sprintf("db_instance_count=%d", t.config.DeploymentConfiguration.DBInstanceCount),
-		"-var", fmt.Sprintf("db_instance_engine=%s", t.config.DeploymentConfiguration.DBInstanceEngine),
-		"-var", fmt.Sprintf("db_instance_class=%s", t.config.DeploymentConfiguration.DBInstanceClass),
-		"-var", fmt.Sprintf("db_username=%s", t.config.DeploymentConfiguration.DBUserName),
-		"-var", fmt.Sprintf("db_password=%s", t.config.DeploymentConfiguration.DBPassword),
-		"-var", fmt.Sprintf("mattermost_download_url=%s", t.config.DeploymentConfiguration.MattermostDownloadURL),
-		"-var", fmt.Sprintf("mattermost_license_file=%s", t.config.DeploymentConfiguration.MattermostLicenseFile),
+		"-var", fmt.Sprintf("cluster_name=%s", t.config.ClusterName),
+		"-var", fmt.Sprintf("app_instance_count=%d", t.config.AppInstanceCount),
```

Comment:
This is another general question. I am wondering if at the end of the deployment process we should automatically create the sysadmin.

Reply:
Yes .. yes .. all that will happen :) I am envisioning it as a one command to deploy and start the load tests. Let me first wrap up the deployment part. That is still a long way to go.